onboard/src/pages/BackgroundInformationPage.jsx

```diff
@@ -1,12 +1,56 @@
 import React from "react";
+import Typography from "@mui/material/Typography";
+import Accordion from "../components/Accordion";
+import PersonalInformation from "./subsections/backgroundInformation/Personalnformation";
+import { isArrayPresent } from "../utils/helperFunctions";
 import "./BackgroundInformationPage.css";
 
+const accordionArr = [
+  {
+    title: "Personal Information",
+    content: <PersonalInformation />,
```

Comment:
Let's 'stub out' little individual components for each section. They can have no content and just be a function that returns `<></>`. This will help us have those modules predefined.

Reply:
I will be working on this